api/v1/verticadb_webhook.go

```diff
@@ -895,6 +896,20 @@ func (v *VerticaDB) hasValidPodSecurityContext(allErrs field.ErrorList) field.Er
 	return allErrs
 }
 
+func (v *VerticaDB) hasValidNMAResourceLimit(allErrs field.ErrorList) field.ErrorList {
+	nmaMemoryLimit := vmeta.GetNMAResource(v.Annotations, v1.ResourceLimitsMemory)
+	if nmaMemoryLimit.IsZero() { // Zero implies it isn't set.
+		return allErrs
+	}
+	if vmeta.MinNMAMemoryLimit.Cmp(nmaMemoryLimit) == 1 {
```

Comment:
Isn't there a min limit for cpu too?

Reply:
No. The memory is the important min to enforce because we cannot run things like catalog editor without at least 1GB of memory.